### DISCUSSION TITLE
$refs not working with stacked Alpine.data()

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#3913] [7. Bugs (archived)]
$refs works for the closest x-data (accordion), but not in the parent x-data (dialog). For details see:
https://codepen.io/simonferndriger/pen/yLGYNOX

![image](https://github.com/alpinejs/alpine/assets/1435963/065883b8-61bb-4228-92b1-4dac7c938b83)

**Comment by SimoTod** [comment]
Yeah, it's expected. it's the same as a variable, the parent can't use a children variable. The other direction is okay.

**Comment by simonferndriger** [comment]
Ah, OK. Thanks for clarifying.
